tutorials/plantUml.md

```diff
@@ -278,3 +278,51 @@ package "Rule Of Thumb";{
 
 Explicitly define all symbols to avoid any potential layout mishaps.
 </box>
+
+### Using reference frames
```

Comment:
Reference frames in PlantUML sequence diagrams allow you to group and reuse sequences of interactions. They help improve diagram readability and reduce repetition in complex scenarios. Here's how to use them effectively:

This could be a intro as suggested by @NeoHW?

Reply:
That's a great suggestion. Added this as a section introduction.